styleguide/source/assets/scss/03-organisms/_sidebar-articles.scss

```diff
@@ -0,0 +1,3 @@
+.ama__sidebar-articles {
+  @include child-top-gutters($gutter / 2);
+}
```

Comment:
Add empty line.

Reply:
Github doesn't seem to be picking up the empty line changes, but I did add them.